manifests/install.pp

```diff
@@ -71,9 +71,17 @@
 
   # if the user didn't specify a version, just use "installed".
   # if they did specify a version, assume they provided a valid one
-  $_version = $version ? {
-    'LATEST' => 'installed',
-    default  => $version,
+  case $::osfamily {
+    'Debian': {
+      $_version = 'installed'
```

Comment:
On 03/12/15, Scott Merrill  <notifications@github.com> wrote:
> 
> In manifests/install.pp(https://github.com/covermymeds/puppet-gluster/pull/26#discussion_r46558910):
>  
> > @@ -71,9 +71,17 @@
> > > # if the user didn't specify a version, just use "installed".
> > # if they did specify a version, assume they provided a valid one
> > - $_version = $version ? {
> > - 'LATEST' => 'installed',
> > - default => $version,
> > + case $::osfamily {
> > + 'Debian': {
> > + $_version = 'installed' 
>  
> Am I reading this correctly that Debian and Ubuntu users will be unable to specify versions to install?
>  
> —
> Reply to this email directly or view it on GitHub(https://github.com/covermymeds/puppet-gluster/pull/26/files#r46558910).
> 
> 
> 
> 
> 
> 
Yes, you are correct. Fixed in next commit.


 

> 
> 
> 
> 
>